keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/SecurityGroupRule.kt

```diff
@@ -24,18 +25,14 @@ sealed class SecurityGroupRule {
   enum class Protocol {
     ALL, TCP, UDP, ICMP
   }
-
-  open val isSelfReference: Boolean = false
```

Comment:
Would need a backreference property for the `SecurityGroupSpec` in order to have a val like this and it was only used in one place, so I just inlined it